### ISSUE TITLE
high voltage bug?

### ISSUE THREAD

**Issue by ladyada** [adafruit/Adafruit_CircuitPython_INA219#10]
Re this thread:  https://forums.adafruit.com/viewtopic.php?f=19&t=153587

User couldn't get the CircuitPython code to read more than 16V, even after setting the measuring range to 32V.  After some testing, they noticed a consistent 16.4V offset between the voltage as measured by a VOM and the output from the microcontroller.

**Comment by barbudor** [comment]
@ladyada , @siddacious 
I think I got it
`raw_bus_voltage         = ROBits( 12, _REG_BUSVOLTAGE, 3, 2, False)`
The bus voltage value is in fact 13 bits from bit 3 to bit 15, not 12.
My mistake. I didn't tested it with voltage over 16V
Beside, it is a unsigned value. Shouldn't I have used `ROUnaryStruct()` instead of `ROBits()` ?

Strangely, the `bus_voltage_range` bit doesn't impact the LSB weight of the ADC which is always 4mv in any case. Selecting a range of 16V seems only to limit the ADC to 12 bit dynamic while setting it to 32V enables the 13th bit.
This is what I understand from 8.6.3.2 in the datasheet

Not sure I can setup a INA and a PSU over 16V today but the problem is clearly here
Can help this week-end probably if you don't have the time to fix it before

Cheers
